edx_notifications/server/web/static/edx_notifications/js/views/counter_icon_view.js

```diff
@@ -132,9 +132,8 @@ var CounterIconView = Backbone.View.extend({
                  notification_alert.play();
              }
              counterView.render();
-             if (counterView.notification_pane) {
-                 counterView.notification_pane.collection.url = counterView.options.endpoints.user_notifications_unread_only;
-                 counterView.notification_pane.hydrate();
+             if (counterView.notification_pane && counterView.notification_pane.selected_pane == "unread") {
```

Comment:
@chrisndodge here I am ensuring that selected pane should be 'unread'. 
Because i was seeing that if a user is under 'view all' tab/pane and if a notification(s) arrives at that time then unread notifications are rendering under 'view all' pane. So i am just rendering the unread notifications under the 'unread' pane.